apps/accounts/verify.py

```diff
@@ -0,0 +1,53 @@
+import logging
+
+import requests
+
+log = logging.getLogger('accounts.verify')
+ProfileNotFound = LookupError
```

Comment:
This is a little risky because `except LookupError` will catch it. If you define it like:
````
class ProfileNotFound(LookupError):
    pass
````
then `except LookupError` will not catch it which I think is safer and more of an expected behavior.